### ISSUE TITLE
Java wrong checksum error

### ISSUE THREAD

**Issue by wklumpen** [r5py/r5py#298]
**Describe the bug**
After performing a clean run execution, I now have the following traceback when submitting travel times:

```
java.io.java.io.IOException: java.io.IOException: Wrong index checksum, store was not closed properly and could be corrupted.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "TransportNetwork.java", line 126, in com.conveyal.r5.transit.TransportNetwork.fromFiles
Exception: Java Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
... other_traceback stuff ...
  File ".../r5py/r5/transport_network.py", line 48, in __init__
    self._transport_network = com.conveyal.r5.transit.TransportNetwork.fromFiles(
java.io.java.io.IOError: java.io.IOError: java.io.IOException: Wrong index checksum, store was not closed properly and could be corrupted.
```

Wondering does this have something to do with not shutting down the java properly?

**Environment:**
 - OS: Ubuntu 22.04.2 LTS
 - Python package source (mamba, with R5py head installed via pip)
 - Versions of Python, Java Development Kit, Python module

**Comment by christophfink** [comment]
That should be fixed in PR #291 - let me finish and merge that and then you could try again :)

**Comment by christophfink** [comment]
> That should be fixed in PR #291 - let me finish and merge that and then you could try again :)

If you try with the current r5py head, this should work

**Comment by fbarbe00** [comment]
I have just come up with this issue again
